charts/castai-cluster-controller/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: castai-cluster-controller
 description: Cluster controller is responsible for handling certain Kubernetes actions such as draining and deleting nodes, adding labels, approving CSR requests.
 type: application
-version: 0.54.1
+version: 1.0.0
```

Comment:
:O :)

Reply:
WDYT, this is a breaking change, so I think 1.0.0 makes sense ?